editoast/src/views/infra/auto_fixes/mod.rs

```diff
@@ -182,6 +206,116 @@ fn fix_infra(
     Ok(operations)
 }
 
+fn reduce_operation(op1: Option<Operation>, op2: Option<Operation>) -> Option<Operation> {
```

Comment:
To insist on order, how about
```suggestion
fn reduce_operation(early_op: Option<Operation>, late_op: Option<Operation>) -> Option<Operation> {
```

Reply:
Done in https://github.com/osrd-project/osrd/pull/6012/commits/55a83060486a352263aefe26538ff857476ad608.